src/models/msgraph/event.entity.ts

```diff
@@ -0,0 +1,22 @@
+export interface Event {
```

Comment:
daca nu ai o clasa care sa fie salvata in DB - nu se numeste entity...eventual e model sau si mai bine cred ca e dto.
Daca trebuie salvata in DB , atunci nu e interfate ci poti defini o clasa Event de exemplu 
`@Table
  class Event extends Model<Event> {
  ...
  }
`
:)

Reply:
Am mutat tot din entity in dto folder